logging/logging_test.go

```diff
@@ -314,7 +314,7 @@ func TestMaxPartCountLog(t *testing.T) {
 	for {
 		select {
 		case result := <-logging.GetLogsDataChannel():
-			if result.ErrorInfo.Message != "" {
+			if result.ErrorInfo != nil && result.ErrorInfo.Message != "" {
```

Comment:
Actually if there is ErrorInfo then message should present. You can check only result.ErrorInfo != nil.

Reply:
done